source/features/confirm-short-comments.tsx

```diff
@@ -5,12 +5,17 @@ import features from '.';
 
 
 function handleIssueComment(event: delegate.Event<MouseEvent, KeyboardEvent, HTMLAnchorElement>): void {
-    let body = document.querySelector('textarea[name="comment[body]"], textarea#issue_body')?.value.length;
+    let textareas = document.querySelector('textarea[name="comment[body]"], textarea#issue_body')?.value.length;
 	let title = document.querySelector('input#issue_title')?.value.length;
```

Comment:
```suggestion
	const textareas = select('textarea[name="comment[body]"], textarea#issue_body')!.value.length;
	const title = select('input#issue_title').value.length;
```

Reply:
Thank you.

Reply:
It says that "Outdated suggestions cannot be applied" so I committed them manually _(I used optional chaining `?` in the second line, as the `title` textarea might not exist in the page)._
Thank you.